docs/test_resource_yamls/daemonset.yaml

```diff
@@ -14,6 +13,8 @@ spec:
       labels:
         name: fluentd-elasticsearch
     spec:
+      nodeSelector:
+        disktype: ssd
```

Comment:
https://chatgpt.com/share/6714e73a-6290-800b-adde-feefaf56d197

Reply:
This was only used to test Node Selector parameter.